test/functional/feature_blocksdir.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test the blocksdir option.
+"""
+
+import os
+import re
```

Comment:
Linter complains `test/functional/feature_blocksdir.py:9:1: F401 're' imported but unused`. Should either do
```suggestion
```
or (partially?) backport https://github.com/bitcoin/bitcoin/pull/12853